jobs/regular/discourse_post_event/bump_topic.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Jobs
+  class DiscoursePostEventBumpTopic < ::Jobs::Base
+    sidekiq_options retry: false
+
+    def execute(args)
+      raise Discourse::InvalidParameters.new(:topic_id) if args[:topic_id].blank?
```

Comment:
```suggestion
      return if Topic.find_by(id: args[:topic_id])
```

We don't handle these exceptions anywhere, so it's better to just do an early return.